src/lib/internal/actions/floating/action.ts

```diff
@@ -38,6 +38,8 @@ const ALIGN_OPTIONS = ['start', 'center', 'end'] as const;
 type Side = (typeof SIDE_OPTIONS)[number];
 type Align = (typeof ALIGN_OPTIONS)[number];
 
+const isFlipOptions = (value: any): value is FlipOptions => typeof value === 'object';
```

Comment:
This type-guard is misleading, and can lead to errors in the future. Wouldn't an `isObject` type guard suffice? IIRC we have one already.

Reply:
Good idea, replaced with `isObject`.